CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ### Fixed
 
+- Fix type errors in strict mode (#664)
```

Comment:
I think it should be clarified that this refers to PHP's strict mode.  The library itself also has a strict mode setting for the CSS (which we plan to remove).

Though I'm not sure this warrants a changelog entry at all.  It's not fixing any actual bug; merely allowing for `declare_strict` to be introduced, with a couple of explicit casts to replace the implicit ones.

Reply:
I've clarified this accordingly.

Reply:
I'd like to keep the changelog entry. This was actually something where the code was incorrect (passing a `float` or `int` where only a `string` was allowed) - only PHP's implicit type conversations had allowed us to get away with this so far. :wink: